docker-compose.yml

```diff
@@ -19,6 +19,17 @@ services:
     expose:
       - "9000"
 
+  redis:
+    image: redis:5 # current stable version, v6 out between March 2020 and May 2020
+    hostname: redis
+    entrypoint: /usr/local/bin/redis-server --appendonly yes --appendfsync everysec # using combination of RDB and AOF for persistence: https://redis.io/topics/persistence
+    volumes:
+      - redis-data:/data
+    networks:
+      - xain-fl-dev
+    ports:
+      - "6379:6379"
```

Comment:
Isn't `expose` better here instead of `ports` (like we did with minio)?

Reply:
Yes! For `docker-compose.yml` at least, for sure. For development file it makes sense for making it easier for DEVs to access Redis directly from the host, but for production environment, it should be kept restricted to local Docker network. Fixing it!